cli/Valet/Site.php

```diff
@@ -398,7 +398,7 @@ public function replaceOldLoopbackWithNew(string $siteConf, string $old, string
 
         $lookups = [];
         $lookups[] = '~#?listen .*:80; # valet loopback~';
-        $lookups[] = '~#?listen .*:443 ssl http2; # valet loopback~';
+        $lookups[] = '~#?listen .*:443 ssl; # valet loopback~';
```

Comment:
Agreed. This should suffice, since the old configs from when nginx used to support the `http2` directive, won't work on current nginx versions anyway.